scripts/generateAssetData/utils/portals.ts

```diff
@@ -1,21 +1,57 @@
 import type { ChainId } from '@shapeshiftoss/caip'
 import { ASSET_NAMESPACE, bscChainId, toAssetId } from '@shapeshiftoss/caip'
-import type { Asset } from '@shapeshiftoss/types'
+import type { Asset, AssetsByIdPartial } from '@shapeshiftoss/types'
+import { isSome } from '@shapeshiftoss/utils'
 import axios from 'axios'
 import qs from 'qs'
 import { getAddress, isAddressEqual, zeroAddress } from 'viem'
 import { CHAIN_ID_TO_PORTALS_NETWORK } from 'lib/market-service/portals/constants'
-import type { GetTokensResponse, TokenInfo } from 'lib/market-service/portals/types'
+import type {
+  GetPlatformsResponse,
+  GetTokensResponse,
+  PlatformsById,
+  TokenInfo,
+} from 'lib/market-service/portals/types'
 
+import generatedAssetData from '../../../src/lib/asset-service/service/generatedAssetData.json'
 import { colorMap } from '../colorMap'
 import { createThrottle } from '.'
 
+const assets = generatedAssetData as unknown as AssetsByIdPartial
+
+export const fetchPortalsPlatforms = async (): Promise<PlatformsById> => {
+  const url = 'https://api.portals.fi/v2/platforms'
+  const PORTALS_API_KEY = process.env.REACT_APP_PORTALS_API_KEY
+  if (!PORTALS_API_KEY) throw new Error('REACT_APP_PORTALS_API_KEY not set')
```

Comment:
nipick - move outside of function so it can throw on import if needed and we have it for any helper functions that might need it

Reply:
https://github.com/shapeshift/web/pull/7473/commits/03968cb60afb5495d36b62742bb58db8bfcc009a and regen as paranoia to confirm this works